api/src/main/java/org/eclipse/microprofile/reactive/messaging/Incoming.java

```diff
@@ -88,34 +90,8 @@
 
     /**
      * The value of the consumed stream.
```

Comment:
This should say "The name of the stream being consumed".  There are some other small changes I would like to put into the doc in text that is not part of this PR so I will do a small PR after this one is merged.

Reply:
I've replaced this with:

```
/**
     * The name of the consumed channel.
     *
     * @return the name of the consumed channel, must not be blank.
     */
```

which is more in line with the content of the spec.